include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,252 @@
+    {
+        using stream_char_t = typename stream_type::char_type;
+        auto stream_view = std::ranges::subrange<decltype(std::istreambuf_iterator<stream_char_t>{stream}),
+                                                 decltype(std::istreambuf_iterator<stream_char_t>{})>
+                           {std::istreambuf_iterator<stream_char_t>{stream},
+                            std::istreambuf_iterator<stream_char_t>{}};
+        auto stream_it = std::ranges::begin(stream_view);
+
+        if (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank), std::string{"LOCUS"})))
+            throw parse_error{"An entry has to start with the code word LOCUS."};
+
+        //ID
+        if constexpr (!detail::decays_to_ignore_v<id_type>)
+        {
+            if (options.embl_genbank_complete_header)
+            {
+                std::ranges::copy(std::string{"LOCUS"}, std::back_inserter(id));
+
+                while (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                                            std::string{"ORIGIN"})))
+                {
+                        std::ranges::copy(stream_view | view::take_line_or_throw
+                                                	  | view::char_to<value_type_t<id_type>>,
+                                     	  std::back_inserter(id));
+                        std::ranges::copy(std::string{"\n"}, std::back_inserter(id));
+                }
+            }
+            else
+            {
+				// ID
+                detail::consume(stream_view | view::take_until(!is_blank));
+                // read id
+                std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank)
+                                              | view::char_to<value_type_t<id_type>>,
+                                  std::back_inserter(id));
+                detail::consume(stream_view | view::take_line_or_throw);
+            }
+        }
+
+        // Jump to sequence
+        while (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank),
+                                    std::string{"ORIGIN"}) | options.embl_genbank_complete_header))
+        {
+            detail::consume(stream_view | view::take_until(is_cntrl));
+            ++stream_it;
+        }
+
+        // Sequence
+        detail::consume(stream_view | view::take_line_or_throw);
+        auto constexpr is_end = is_char<'/'> ;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+        {
+            auto constexpr is_legal_alph = is_in_alphabet<seq_legal_alph_type>;
+            std::ranges::copy(stream_view | ranges::view::remove_if(is_space || is_digit)
+                                          | view::take_until_or_throw_and_consume(is_end) // consume //
+                                          | std::view::transform([is_legal_alph] (char const c) // enforce legal alphabet
+                                         {
```

Comment:
fix indentation. this belongs to the transform view and should not be aligned with the pipe symbol